### PR TITLE
fix(schema): generate 'any' instead of '{}'

### PR DESCRIPTION
**Pull request by melloware**
## Status
**READY**

Fix #321 

## Description
Instead of generating an empty type like...

```ts
export type Health200 = {};
```

it generates it as an ANY

```ts
export type Health200 = any;
```

@CPatchane @anymaniax Can you review this it looks to me like the first `if` and the last `else` were doing the exact same thing so I simplified the logic "I think".

### CODE REVIEW

**Comment by melloware** [comment]
Ahh nevermind I see the logic now that it broke a unit test the else's were not in the same if.

**Comment by melloware** [comment]
Fixed https://github.com/anymaniax/orval/issues/308

File: src/core/generators/schemaDefinition.ts [hunk 1]
```diff
@@ -71,22 +71,25 @@ export const generateSchemasDefinition = async (
...
+            output += `export type ${schemaName} = ${outputValue};\n`;
           } else {
-            const alias = imp?.specKey
+            let alias = imp?.specKey
               ? `${pascal(getSpecName(imp.specKey, context.specKey))}${
                   resolvedValue.value
                 }`
               : `${resolvedValue.value}Bis`;
 
+            alias = alias === '{}' ? 'any' : alias;
```
Comment (anymaniax) [inline src/core/generators/schemaDefinition.ts]: will never be `{}` in that case

Comment (CPatchane) [reply src/core/generators/schemaDefinition.ts]: I don't think you need to change the alias here, just the output value should be changed or am I wrong?

Comment (anymaniax) [reply src/core/generators/schemaDefinition.ts]: In that case, that will be always a schema so it's always a name

**Comment by melloware** [comment]
OK updated and remove the `alias` part.

**Pull request merged**